scripts/convert_to_onnx.py

```diff
@@ -1,22 +1,55 @@
 import torch
+import torch.nn as nn
 from pathlib import Path
 import argparse
+from mann_pytorch.utils import read_from_file
+import numpy as np
 
 
-def convert_model(model_path: Path, onnx_model_path: Path, opset_version: int):
+def convert_model(model_path: Path, onnx_model_path: Path, normalization_folder: Path, opset_version: int):
     # Restore the model with the trained weights
     mann_restored = torch.load(str(model_path))
 
     # Set dropout and batch normalization layers to evaluation mode before running inference
     mann_restored.eval()
+    input_size = next(mann_restored.parameters()).size()[1]
+
+    # Here we create two layes for normalization and denormalization
+    X_mean = read_from_file(str(normalization_folder / "X_mean.txt"))
+    X_std = read_from_file(str(normalization_folder / "X_std.txt"))
+    X_std[np.where(X_std <= 1e-4)] = 1
```

Comment:
This is how I was neglecting zero variances up to now, but after reading @S-Dafarra's comment [here](https://github.com/ami-iit/bipedal-locomotion-framework/pull/652#discussion_r1169822799) I guess that using the  threshold (1e-4 here) is actually more appropriate.

We should update the normalization also at training time though, to use it consistently at inference time.

Reply:
> We should update the normalization also at training time though, to use it consistently at inference time.

Let's check this at the next training attempt, but merge this PR in the meanwhile. We will integrate these changes in a new PR if needed.